src/network/networkd-link.c

```diff
@@ -3931,10 +3916,6 @@ static int link_carrier_gained(Link *link) {
         if (r < 0)
                 return r;
 
-        r = link_set_bridge_mdb(link);
-        if (r < 0)
-                return r;
-
```

Comment:
I did put the logic here because MDB entries cannot be configured on interfaces without a carrier.

With the code as it stands, I see the following:

```
enp57s0u2u2: Gained IPv6LL
testbr: Gained IPv6LL
Enumeration completed
testbr: netdev exists, using existing without changing its parameters
testbr: IPv6 successfully enabled
testbr: Link DOWN
enp0s20f0u1u1: Could not add MDB entry: Invalid argument
enp0s20f0u1u1: Failed
```

Once `enp0s20f0u1u1` gains a carrier, the MDB entries are in fact configured correctly, so one could argue it's a cosmetic issue. I would still keep the code in `link_carrier_gained()`.